spring-cloud-gcp-autoconfigure/src/test/resources/logback-test.xml

```diff
@@ -23,6 +23,8 @@
 		<appender-ref ref="STDOUT" />
 	</logger>
 
+	<property name="GOOGLE_CLOUD_PROJECT" value="test-project" />
```

Comment:
I've just copied/pasted this code into my local (fresh) copy of the project and nothing changed when I run `GcpStorageAutoConfigurationTests`:
```
Caused by: java.lang.IllegalStateException: Logback configuration error detected: 
ERROR in ch.qos.logback.core.joran.spi.Interpreter@15:14 - RuntimeException in Action for tag [appender] java.lang.IllegalArgumentException: A project ID is required for this service but could not be determined from the builder or the environment.  Please set a project ID using the builder.
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadConfiguration(LogbackLoggingSystem.java:166)
```

Am I missing anything else?

Thanks